intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -683,7 +683,10 @@ \section{Transfer Files to/from the \hpc}
   For example, we will copy the (local) file ``\emph{localfile.txt}'' to your
   home directory on the \hpc cluster. We first generate a small dummy
   ``\emph{localfile.txt}'', which contains the word ``Hello''.  Use your own
-  $<$vsc-account$>$, which is something like ``\emph{\userid}''.
+  $<$vsc-account$>$, which is something like ``\emph{\userid}''. Don't
+  forget the colon at the end: if you forget it, it will just create a file
+  named \texttt{\userid{}@\loginnode{}} on your local filesystem. You can even
+  specify where to save the file by putting a path after the colon.
```

Comment:
"where to save the file on the remote file system" + show an example of the latter

are we already explaining both directions (from workstation to HPC & vice versa)?